webapp/tests/data_access/test_form_data_controller.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+from unittest.mock import patch
+
+import fakeredis as fakeredis
+
+from pydantic import MissingError
+from app.data_access.form_data_controller import save, get
+from app.config import Config
+
+redis_fake = fakeredis.FakeStrictRedis()
+
+
+@patch("app.data_access.form_data_controller.r", redis_fake)
+class TestFormDataController(unittest.TestCase):
```

Comment:
This is probably a problem with the current setup. Some of the tests still are unit tests, while others are pytest tests. We wanted to use pytests for new tests because it is easier to read and adds fixtures which help a lot in keeping complexity out of tests.

Maybe you could rewrite those tests as pytests?